AngularJS/angularjs-quiz.md

```diff
@@ -1,199 +1,226 @@
-    <h2>Genders</h2>
-    <div \*ngFor="let user of users | async">{{ user.gender }}</div>
-```
-    - [] None. The async pipe does not subscribe automatically. None. The null pipe does not subscribe automatically.
-    - [] None. The template syntax is not correct. None. The template syntax is not correct.
-       v2:{5};
-    - [] Three. There is one for each async pipe. Three. There is one for each null pipe.
-    - [] One. The async pipe caches Observables by type internally.
+- [] None. The async pipe does not subscribe automatically. None. The null pipe does not subscribe automatically.
+- [] None. The template syntax is not correct. None. The template syntax is not correct.
+v2:{5};
+- [] Three. There is one for each async pipe. Three. There is one for each null pipe.
+- [] One. The async pipe caches Observables by type internally.
+
 
 #### Q5. How can you use the HttpClient to send a POST request to an endpoint from within an addOrder function in this OrderService?
 
+```javascript
+export class OrderService {
+constructor(private httpClient: HttpClient) { }
+addOrder(order: Order) {
+// Missing line
+}
+}
 ```
-    export class OrderService {
-    constructor(private httpClient: HttpClient) { }
+- [] this.httpClient.url(this.orderUrl).post(order);null
+- [] this.httpClient.send(this.orderUrl, order);null
+- [] this.httpClient.post<Order>(this.orderUrl, order);null
+- [] this.httpClient.post<Order>(this.orderUrl, order) .subscribe();
 
-        addOrder(order: Order) {
-            // Missing line
-        }
-        }
-```
-    - [] this.httpClient.url(this.orderUrl).post(order);null
-    - [] this.httpClient.send(this.orderUrl, order);null
-    - [] this.httpClient.post<Order>(this.orderUrl, order);null
-    - [] this.httpClient.post<Order>(this.orderUrl, order) .subscribe();
 
 #### Q6. What is the RouterModule.forRoot method used for? What is the null method used for?
+- [] registering any providers that you intend to use in routed components registering any providers that you intend to use in routed components
+- [] registering route definitions at the root application level registering route definitions at the root application level
+- [] indicating that Angular should cheer on your routes to be successful indicating that Angular should cheer on your routes to be successful
+- [] declaring that you intend to use routing only at the root level
 
-    - [] registering any providers that you intend to use in routed components registering any providers that you intend to use in routed components
-    - [] registering route definitions at the root application level registering route definitions at the root application level
-    - [] indicating that Angular should cheer on your routes to be successful indicating that Angular should cheer on your routes to be successful
-    - [] declaring that you intend to use routing only at the root level
 
 #### Q7. Which DOM elements will this component metadata selector match on?
 
+```javascript
```

Comment:
can please try to use angular instead of javascript which is more related to this quiz?